Sources/tart/Commands/Push.swift

```diff
@@ -23,6 +23,9 @@ struct Push: AsyncParsableCommand {
                              """))
   var chunkSize: Int = 0
 
+  @Flag(help: .hidden)
+  var oldDiskFormat: Bool = false
```

Comment:
I have some ideas for even further improvements so we might change the format again in a year or so. Let's make it a vertion thing like `v1`, `v2`, etc. This way it will be easier for us to modify Pakcer builds to still build ventura images using v1 and only sonoma using v2.

Reply:
Implemented in https://github.com/cirruslabs/tart/pull/589/commits/e529472ad668ce56e71431d9ad86a991e1d10307.